pages/common/logstash.md

```diff
@@ -0,0 +1,12 @@
+# logstash
+
+> Migrate data from one source into another.
+> Usually used to ingest data from various other data sources into elasticsearch.
+
+- Check validity of a logstash configuration:
+
+`logstash --configtest --config logstash-config.conf`
+
+- Run logstash using configuration:
+
+`sudo logstash --config logstash-config.conf`
```

Comment:
Is sudo not needed in the most common use case? If it is, then we *should* include it, per [guideline #2](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines)